addons/l10n_es_pos/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="point_of_sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//block[@id='pos_accounting_section']" position="inside">
+                <field name="pos_is_spanish" invisible="1" />
+                <setting string="Simplified Invoice Limit" title="" attrs="{'invisible': [('pos_is_spanish','=',False)]}">
+                    <div class="text-muted">
+                        Above this limit the simplified invoice won't be made
+                    </div>
+                    <field name="pos_l10n_es_simplified_invoice_limit" />
```

Comment:
I miss being able to set the simplified invoice sequence from the settings, which is a usual need when the default doesn't fit the user preferences.

Reply:
You can still change the sequence from the settings: 
![image](https://github.com/odoo/odoo/assets/118446179/0f2f335c-0a81-4cca-ac5e-f6fc72983bae)
Do you think it would be better to have a specific setting for this in the pos config?  I figured that it's better to use the default settings for sequences

Reply:
Ok, not a big deal :)